### PR TITLE
added redhat courses

### PR DESCRIPTION
**Pull request by SyedAahana**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by SyedAahana** [comment]
i have added the redhat courses which are free

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -848,6 +848,11 @@
 * [React Native Tutorial (2021)](https://youtube.com/playlist?list=PL8kfZyp--gEXs4YsSLtB3KqDtdOFHMjWZ) - Programming with Mash, MAhdi SHarifimehr (YouTube)
 * [React Tutorial for Beginners](https://www.youtube.com/playlist?list=PLC3y8-rFHvwgg3vaYJgHGnModB54rxOk3) - codevolution
```
Comment (shrn01) [inline courses/free-courses-en.md:850]: Please add 2 new lines after each section

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -848,6 +848,11 @@
...
 
+### Redhat
+
+* [Fundamentals of Red Hat Enterprise Linux](https://www.edx.org/course/fundamentals-of-red-hat-enterprise-linux) - edX
```
Comment (shrn01) [inline courses/free-courses-en.md:853]: Move it into Linux

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -848,6 +848,11 @@
...
+* [Fundamentals of Containers, Kubernetes, and Red Hat OpenShift](https://www.edx.org/course/fundamentals-of-containers-kubernetes-and-red-hat) - edX
```
Comment (shrn01) [inline courses/free-courses-en.md:854]: Move to kubernetes

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -848,6 +848,11 @@
...
+* [Fundamentals of Java EE Development](https://www.edx.org/course/fundamentals-of-java-ee-development) - edX
```
Comment (shrn01) [inline courses/free-courses-en.md:855]: Move to java

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: courses/free-courses-en.md [hunk 5]
```diff
@@ -694,6 +694,7 @@
 * [Advanced Software Construction in Java](https://openlearninglibrary.mit.edu/courses/course-v1:MITx+6.005.2x+1T2017/about) - MIT Open Learning Library
 * [Central Connecticut State University, Introduction to CS Using Java](http://chortle.ccsu.edu/CS151/cs151java.html)
 * [CS106A - Programming Methodology](https://see.stanford.edu/Course/CS106A) (Stanford)
+* [Fundamentals of Java EE Development](https://www.edx.org/course/fundamentals-of-java-ee-development) - edX
```
Comment (eshellman) [inline courses/free-courses-en.md:697]:
```suggestion
* [Fundamentals of Java EE Development](https://www.edx.org/course/fundamentals-of-java-ee-development) -  Will Dinyes (edX)
```

File: courses/free-courses-en.md [hunk 6]
```diff
@@ -903,12 +904,14 @@
 
 ### Kubernetes
 
+* [Fundamentals of Containers, Kubernetes, and Red Hat OpenShift](https://www.edx.org/course/fundamentals-of-containers-kubernetes-and-red-hat) - edX
```
Comment (eshellman) [inline courses/free-courses-en.md:907]:
```suggestion
* [Fundamentals of Containers, Kubernetes, and Red Hat OpenShift](https://www.edx.org/course/fundamentals-of-containers-kubernetes-and-red-hat) - Zach Gutterman, Richard Allred (edX)
```

File: courses/free-courses-en.md [hunk 7]
```diff
@@ -903,12 +904,14 @@
...
 ### Kubernetes
 
+* [Fundamentals of Containers, Kubernetes, and Red Hat OpenShift](https://www.edx.org/course/fundamentals-of-containers-kubernetes-and-red-hat) - Zach Gutterman, Richard Allred (edX)
 * [Kubernetes Core Concepts](https://kube.academy/paths/kubernetes-core-concepts) - KubeAcademy (VMware)
 * [Kubernetes Tutorial for Beginners](https://www.youtube.com/playlist?list=PLy7NrYWoggjziYQIDorlXjTvvwweTYoNC) - TechWorld with Nana
 
 
 ### Linux
 
+* [Fundamentals of Red Hat Enterprise Linux](https://www.edx.org/course/fundamentals-of-red-hat-enterprise-linux) - edX
```
Comment (eshellman) [inline courses/free-courses-en.md:914]:
```suggestion
* [Fundamentals of Red Hat Enterprise Linux](https://www.edx.org/course/fundamentals-of-red-hat-enterprise-linux) - Chris Caillouet (edX)
```

**Pull request merged**